auto-setup.Dockerfile

```diff
@@ -0,0 +1,70 @@
+ARG BASE_BUILDER_IMAGE=temporalio/base-builder:1.4.0
+ARG BASE_SERVER_IMAGE=temporalio/base-server:1.5.0
+ARG GOPROXY
+
+##### Builder #####
+FROM ${BASE_BUILDER_IMAGE} AS temporal-builder
+
+WORKDIR /home/builder
+
+# cache Temporal packages as a docker layer
+COPY ./temporal/go.mod ./temporal/go.sum ./temporal/
+RUN (cd ./temporal && go mod download)
+
+# cache tctl packages as a docker layer
+COPY ./tctl/go.mod ./tctl/go.sum ./tctl/
+RUN (cd ./tctl && go mod download)
+
+# build
+COPY . .
+RUN (cd ./temporal && make bins)
+RUN (cd ./tctl && make build)
+
+##### Temporal server #####
+FROM ${BASE_SERVER_IMAGE} as temporal-server
```

Comment:
If you are creating dedicated `Dockerfile` for auto-setup, you don't need this intermediate layer here.

Reply:
done